modules/node-shared/src/main/scala/org/tessellation/node/shared/infrastructure/snapshot/CurrencySnapshotCreator.scala

```diff
@@ -82,7 +84,8 @@ object CurrencySnapshotCreator {
       trigger: ConsensusTrigger,
       events: Set[CurrencySnapshotEvent],
       rewards: Option[Rewards[F, CurrencySnapshotStateProof, CurrencyIncrementalSnapshot, CurrencySnapshotEvent]],
-      facilitators: Set[PeerId]
+      facilitators: Set[PeerId],
+      maybeFeeTransactions: Option[SortedSet[Signed[FeeTransaction]]] = None
```

Comment:
either `feeTransactions` or change `rewards` to `maybeRewards`. Please don't mix it (same above).

Reply:
@marcinwadon I was thinking of renaming it to `fallbackFeeTransactions`. WDYT?